Sources/TCACoordinators/TCARouter/TCARouter.swift

```diff
@@ -34,16 +34,16 @@ public struct TCARouter<
       }
     )
   }
-
+  
   public var body: some View {
     Router(
-      viewStore.binding(get: routes, send: updateRoutes),
+      viewStore.binding(get: { _ in store.withState(routes) }, send: updateRoutes),
```

Comment:
In my testing, this still had stale data, but the following always had the latest data:
```suggestion
      ViewStore(store, observe: { $0 }).binding(get: routes, send: updateRoutes),
```